qa/common/common.sh

```diff
@@ -0,0 +1,199 @@
+#
```

Comment:
Can't we just use python and a proper test framework (eg. being able to call just a single test, list available tests, mark slow running tests,...)?

Reply:
We can if someone else does the work :-)

Reply:
We could also turn the question around: instead of using Python and a complicated test framework, can't we just use a bash script? This PR is my answer to that question.

Reply:
Please note that upstream has a Python-based testing framework called `teuthology` and the upstream developers have already written:

* a `cephadm` task for deploying Ceph with cephadm
* a `rados/cephadm` suite for testing core Ceph components in environments deployed using cephadm

So I think the need for integration testing via a complicated Python-based framework is already being met upstream, and we will be running those tests downstream in environments prepared by ceph-bootstrap.